warden/app/app_config.go

```diff
@@ -189,6 +189,7 @@ var (
 		{Account: ibctransfertypes.ModuleName, Permissions: []string{authtypes.Minter, authtypes.Burner}},
 		{Account: ibcfeetypes.ModuleName},
 		{Account: icatypes.ModuleName},
+		{Account: intentmoduletypes.ModuleName},
```

Comment:
Do I understand correctly that there is no private key for the module account?

Reply:
correct, modules have a special type of account called "ModuleAccount" which is just an abstraction for executing on-chain code or programmatically holding tokens - in a way, they're similar to ethereum smart contracts addresses